catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/ResourceOperations.java

```diff
@@ -99,6 +99,12 @@
 
   private static final String QUERY_RETURNED_NULL = "Query returned null metacard";
 
+  private static final String GET_ENTERPRISE_RESOURCE_OPTIONS = "getEnterpriseResourceOptions";
```

Comment:
✏️ Think you missed one spot on line 142: https://github.com/codice/ddf/blob/38faa85e387b86e9804e9a1070670c3290a904dc/catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/ResourceOperations.java#L142